### ISSUE TITLE
mask head usage?

### ISSUE THREAD

**Issue by Scitator** [ignacio-rocco/detectorch#7]
Hi!

Firstly, thank u a lot. It looks like the best mask-rcnn implementation. (cause it does not use this magic `from config import cfg`)

Nevertheless, something is a bit strange for me. In the model `init` you define `self.mask_head` (cause it needed for segmentation task), but in the `forward` pass you dont use it at all. How is that?

Thanks!

**Comment by ignacio-rocco** [comment]
Hi, glad you liked the code :)

Please see the demo notebooks for the usage of the mask head. Essentially there is a results postprocess stage between the forward pass and running the mask head. 

Ideally this post-processing stage (which involves NMS and other things) should also be converted into a  torch.nn.Module, but this is not the case for now.
